src/test/transaction_tests.cpp

```diff
@@ -821,9 +821,29 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
     BOOST_CHECK(!IsStandardTx(CTransaction(t), reason));
     BOOST_CHECK_EQUAL(reason, "scriptsig-size");
 
+    // Check tx-size (non-standard if transaction weight is > MAX_STANDARD_TX_WEIGHT)
+    t.vin.clear();
+    t.vin.resize(2438); // size per input (empty scriptSig): 41 bytes
+    t.vout[0].scriptPubKey = CScript() << OP_RETURN << std::vector<unsigned char>(19, 0); // output size: 30 bytes
+    // tx header:                12 bytes =>     48 vbytes
```

Comment:
micro-nit: I don't think this big block of text is necessary for the test reader. We trust that `GetTransactionWeight` does the right thing, and checking that `IsStandardTx` does what we expect.

Reply:
no need to invalidate ACKs for this, just noting!

Reply:
The idea was to reason how we end up magically from two numbers (2438 and 19) to a transaction size with exactly 400000 vbytes. In case this needs to be altered in the future to reach another tx weight, a test writer has it easier to figure out what to change (I admit that it is very unlikely to ever happen). I agree that it could be a bit shorter though.